components/Navbar.tsx

```diff
@@ -184,28 +189,37 @@ const routes: NavSection[] = [
 
 export const Navbar: React.FC = () => {
   const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false)
+  const [openSubmenus, setOpenSubmenus] = useState<string[]>([]); // Array to track open submenus
 
   const toggleMobileMenu = () => {
     setIsMobileMenuOpen(!isMobileMenuOpen)
   }
 
+  const toggleSubmenu = (sectionName: string) => {
+    if (openSubmenus.includes(sectionName)) {
+      setOpenSubmenus(openSubmenus.filter(name => name !== sectionName));
+    } else {
+      setOpenSubmenus([...openSubmenus, sectionName]);
+    }
+  };
+
   return (
     <header className={`sticky top-0 z-50`}>
       <nav className="bg-blue-navbar flex px-8 justify-between">
         {/* CCV Logo */}
-        <div className="flex items-center py-4 lg:py-8">
+        <div className="flex items-center py-4 lg:py-8 px-12">
           <Link href={"/"}>
             <CCVLogo width={120}/>
           </Link>
         </div>
 
         {/* Navigation Menu for Desktop */}
-        <NavigationMenu.Root className="hidden lg:flex relative z-10 w-screen justify-between items-stretch">
+        <NavigationMenu.Root className="xl:flex hidden relative z-10 w-screen justify-between items-stretch">
```

Comment:
hmm... i see you have `xl:flex` here.

For large screens it should be find to see the whole navbar. Would that be as simple as adding in `lg:flex` here? There might be a few other places to add

Reply:
i changed the breakpoint to be lg... which meant also making the gap between navbar items smaller at medium-ish viewport sizes

Reply:
![Uploading Screenshot 2025-05-20 at 4.49.33 PM.png…]()

Reply:
<img width="955" alt="Screenshot 2025-05-20 at 5 02 51 PM" src="https://github.com/user-attachments/assets/44396980-c627-4fef-91c1-186111eadbe8" />